clang/tools/clang-format/clang-format.el

```diff
@@ -216,17 +327,72 @@ uses the function `buffer-file-name'."
             (if incomplete-format
                 (message "(clang-format: incomplete (syntax errors)%s)" stderr)
               (message "(clang-format: success%s)" stderr))))
-      (delete-file temp-file)
+      (ignore-errors (delete-file temp-file))
```

Comment:
suggest with-demoted-errors here too, could be a function as it's used elsewhere.